IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -129,6 +132,7 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
         if (accessTokenError)
         {
             completionBlock(nil, accessTokenError);
+            self.currentRequestTelemetry.tokenCacheRefresh = NoCachedAT;
```

Comment:
This doesn't exactly match the description: 

` •    CacheMiss = 2, request goes to ESTS because cache entry for the requested token does NOT exist.`

Here it could be that token existed but lookup failed due to a keychain error.

Reply:
Ok i will update to check for accessToken null